examples/kotlin-diktat/run.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
 KTLINT_VERSION=0.39.0
-DIKTAT_VERSION=0.6.3
-SAVE_VERSION=0.1.0-alpha.2
+DIKTAT_VERSION=1.0.0-rc.1
```

Comment:
You can also change diktat version in `buildSrc/build.gradle.kts`; I think there haven't been any new rules or breaking changes since `0.6.1`.

Reply:
Yeah, thanks for the reminder